utils/network.py

```diff
@@ -0,0 +1,282 @@
+"""
+Collects extended network metrics.
+"""
+# stdlib
+import re
+
+# project
+from checks import AgentCheck
+
+GAUGE = AgentCheck.gauge
+MONOTONIC_COUNT = AgentCheck.monotonic_count
+
+SNMP_METRICS = {
+    'Tcp': {
+        # The algorithm used to determine the timeout value used for
+        # retransmitting unacknowledged octets.
+        'RtoAlgorithm': ('tcpx.rto_algorithm', GAUGE),
```

Comment:
We use `tcpx` only because these metrics are not in a release of dd-agent yet. Our preference would be for these to just be under `system.net.tcp` and `docker.net.tcp`.